recipe-server/client/control/components/MultiPicker.js

```diff
@@ -51,88 +44,65 @@ export default class MultiPicker extends React.Component {
       value,
     } = this.props;
 
-    const selectedOptions = options.filter(option =>
-      value.indexOf(option.value) === -1);
-
-    let displayedOptions = [].concat(selectedOptions)
-      .map(option => option.value)
-      .map(val =>
-        (!val ? null : { ...this.convertValueToObj(val) }))
-      .filter(x => x);
-
     const {
       filterText,
     } = this.state;
 
-    if (filterText) {
-      displayedOptions = displayedOptions.filter(option =>
-        JSON.stringify(option).indexOf(filterText) > -1);
-    }
+    let selectedOptions = options.filter(option => value.indexOf(option.value) === -1);
```

Comment:
`value` is actually a string from redux-form, so we're using `indexOf` to search over it. We could split the comma-delineated value and use `includes` on that, but I don't necessarily see the benefit of adding that extra step.